### PR TITLE
Improve GitHub authentication documentation

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/c2cwsgiutils.rst [hunk 1]
```diff
@@ -23,26 +23,24 @@ To have basic authentication of those tools (and on tile generation) you can set
 You can also activate the authentication on GitHub with OAuth2, the configuration is in the
 ``env.project`` file:
 
-* Verify that the ``C2C_AUTH_GITHUB_REPOSITORY`` is correctly configured.
+* Verify that the ``C2C_AUTH_GITHUB_REPOSITORY`` is correctly configured to the project repository.
```
Comment (jwkaltz) [inline doc/integrator/c2cwsgiutils.rst:26]: suggestion:
to the project repository -> to match the project repository name on GitHub.

File: doc/integrator/c2cwsgiutils.rst [hunk 2]
```diff
@@ -23,26 +23,24 @@ To have basic authentication of those tools (and on tile generation) you can set
...
 * By default, the ``C2C_AUTH_GITHUB_ACCESS_TYPE`` is set to admin then only admins users of the previously
-  defined repository have access.
-* You should ask Camptocamp for a new secret and set it in ``C2C_AUTH_GITHUB_CLIENT_SECRET``.
-  It will be defined in the GitHub organizations settings, on 'Developer settings', 'OAuth Apps',
-  'Geoservicies` application, 'Generate a new client secret',
-  the URL should also be configured in the redirect proxy.
+  defined repository have access, can be ``pull``, ``push`` or ``admin``.
 * Define the ``C2C_AUTH_GITHUB_SECRET`` randomly.
 
+For independent applications:
```
Comment (jwkaltz) [inline doc/integrator/c2cwsgiutils.rst:31]: I suppose you mean for applications whose code is not hosted on GitHub?

**Review by jwkaltz — APPROVED** [review]

File: doc/integrator/c2cwsgiutils.rst (hunk 2 above)
Comment (sbrunner) [reply doc/integrator/c2cwsgiutils.rst:31]: No, it's about the GitHub application, I updated the pull request :-)

File: doc/integrator/c2cwsgiutils.rst [hunk 3]
```diff
@@ -23,26 +23,25 @@ To have basic authentication of those tools (and on tile generation) you can set
...
 * By default, the ``C2C_AUTH_GITHUB_ACCESS_TYPE`` is set to admin then only admins users of the previously
-  defined repository have access.
-* You should ask Camptocamp for a new secret and set it in ``C2C_AUTH_GITHUB_CLIENT_SECRET``.
-  It will be defined in the GitHub organizations settings, on 'Developer settings', 'OAuth Apps',
-  'Geoservicies` application, 'Generate a new client secret',
-  the URL should also be configured in the redirect proxy.
+  defined repository have access, can be ``pull``, ``push`` or ``admin``.
 * Define the ``C2C_AUTH_GITHUB_SECRET`` randomly.
 
+For project who create his GitHub applications:
```
Comment (jwkaltz) [inline doc/integrator/c2cwsgiutils.rst:31]: For projects that create their GitHub applications:

**Review by jwkaltz — APPROVED** [review]

**Pull request merged**